src/main/kotlin/org/simplemc/simplehealthbars2/DamageListener.kt

```diff
@@ -40,8 +40,8 @@ class DamageListener(
         }
 
         val healthbar = when (target) {
-            is Player -> playerHealthbar
-            else -> mobHealthbar
+            is Player -> playerHealthbars[target.world.name] ?: playerHealthbars[null]
```

Comment:
Note for reviewing: world configs are keyed by world name, with `null` being the global default.